app/rtc_s1.py

```diff
@@ -872,20 +876,32 @@ def run(cfg: RunConfig):
                         f'{imagery_extension}')
                 output_burst_imagery_list.append(geo_burst_pol_filename)
 
-            _separate_pol_channels(geo_burst_filename,
-                                   output_burst_imagery_list,
-                                   logger, output_raster_format)
+            if os.path.exists(geo_burst_filename):
+                _separate_pol_channels(geo_burst_filename,
+                                       output_burst_imagery_list,
+                                       logger, output_raster_format)
+            else:
+                for filename in output_burst_imagery_list:
+                    os.rename(filename.replace(output_dir, scratch_path), filename)
+
+                # create a vrt from the separated pol tiffs
+                geo_burst_filename +='.vrt'
+                output_imagery_list[-1] = geo_burst_filename
+                gdal.BuildVRT(geo_burst_filename, output_burst_imagery_list,
+                              options=vrt_options_mosaic)
```

Comment:
Sorry, what is the purpose of this code? Was it added by mistake?

Reply:
This was to mimic the output from geocode, but now it is not relevant for this single-threaded code. I will remove the code here.